src/main.ts

```diff
@@ -40,11 +52,11 @@ async function run(): Promise<void> {
     const owner = github.context.repo.owner;
     const repo = payload.repository.name;
 
-    const issue_number = payload.pull_request.number;
-    const issue_title = payload.pull_request.title;
-    const headref = payload.pull_request.head.sha;
-    const baseref = payload.pull_request.base.sha;
-    const labels = payload.pull_request.labels;
+    const pr = payload.pull_request;
```

Comment:
Maybe rename this to `original_pr` or something like that. Further below it gets confusing which PR is meant

Reply:
I like the idea, but went for `mainpr` which is a bit shorter and causes less line wrappings.